### PR TITLE
 added link for ripgrep for Ubuntu Xenial

### PR DESCRIPTION
**Pull request by Rajpratik71**

### CODE REVIEW

File: .whitesource [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "generalSettings": {
+    "shouldScanRepo": true
+  },
+  "checkRunSettings": {
+    "vulnerableCheckRunConclusionLevel": "failure"
+  }
+}
```
Comment (BurntSushi) [inline .whitesource]: Why is this here?

File: README.md [hunk 2]
```diff
@@ -305,6 +305,15 @@ packaging as Debian:
...
 
+If you're an **Ubuntu Xenial (16.04)** user, ripgrep source is
+[available](https://launchpad.net/ubuntu/+source/rust-ripgrep) and the packaging 
+as Debian package is available in [PPAs](https://launchpad.net/~jerem-ferry/+archive/ubuntu/rust) .
+So , add the custom PPA and install it as :
+
+```
+$ sudo apt-get install ripgrep
+```
+
```
Comment (BurntSushi) [inline README.md]: Please fix the formatting here. You have whitespace around punctuation and lines exceeding 79 columns.

File: README.md [hunk 3]
```diff
@@ -305,6 +305,15 @@ packaging as Debian:
 $ sudo apt-get install ripgrep
 ```
 
+If you're an **Ubuntu Xenial (16.04)** user, ripgrep source is
+[available](https://launchpad.net/ubuntu/+source/rust-ripgrep) and the packaging 
+as Debian package is available in [PPAs](https://launchpad.net/~jerem-ferry/+archive/ubuntu/rust) .
```
Comment (BurntSushi) [inline README.md]: This doesn't read right to me? "... and a Debian package is available in a PPA."

File: README.md [hunk 4]
```diff
@@ -305,6 +305,15 @@ packaging as Debian:
...
+So , add the custom PPA and install it as :
```
Comment (BurntSushi) [inline README.md]: Please include *all* commands necessary.

File: .whitesource (hunk 1 above)
Comment (Rajpratik71) [reply .whitesource]: it is a configuration file for Whitesource. , "whitesource" is tool which check for vulnerability in  repositories

Comment (BurntSushi) [reply .whitesource]: Why did you mark this as resolved? I didn't sign off on this. Please remove this from this PR.

I went to their [web site](https://www.whitesourcesoftware.com), and I got immediately spammed with blinking messages. I found no obvious high level description of how it works. I'm immediately skeptical of this tool.

It is poor decorum on your part to try sneaking stuff like this into unrelated PRs. Please don't do it again on any of my repositories.

File: README.md [hunk 5]
```diff
@@ -300,8 +300,17 @@ $ sudo apt-get install ripgrep
...
 [available](https://launchpad.net/ubuntu/+source/rust-ripgrep) using the same
 packaging as Debian:
+```
+$ sudo apt-get install ripgrep
+```
 
+If you're an **Ubuntu Xenial (16.04)** user, ripgrep source is
+[available](https://launchpad.net/ubuntu/+source/rust-ripgrep) and the packaging 
+as Deb package is available in [PPAs](https://launchpad.net/~jerem-ferry/+archive/ubuntu/rust) .
+So , add the custom PPA and install it as :
```
Comment (BurntSushi) [inline README.md]: You still have spaces around punctuation, please remove them.

You still have lines exceeding 79 columns. Please fix that.

Please change `Deb` back to Debian`.

**Comment by BurntSushi** [comment]
You deleted the whitesource config file... and then added it right back.

There are still other lingering issues, but I'm not interested in doing this back-and-forth with you again. Additionally, since the PPA isn't updated, I don't think it's a good idea to add it to the README.
